hail/python/hail/docs/change_log.md

```diff
@@ -24,6 +24,23 @@ an earlier version of Hail to read files written in a later version.
 
 ---
 
+## Version 0.2.102
+
+Released 2022-10-06
+
+### New Features
+
+- (hail#12218) Support missing values in primitive columns in `Table.to_pandas`.
+- (hail#12254) Replaced cross-product-style legends for data groups with factored ones (consistent with `ggplot2`'s implementation) for `hail.ggplot.geom_point`, and added support for custom legend group labels.
+- (hail#12268) `VariantDataset` now implemented `union_rows` for combining datasets with the same samples but disjoint variants.
```

Comment:
Implements

Reply:
Fixed, and made tenses consistent for the other entries as well!